addons/Broseem/Broseem.lua

```diff
@@ -0,0 +1,187 @@
+    [3] = 'dark matter',
+}
+
+local paths = {
+    [8] = 'melee',
+    [264] = 'ranged',
+    [520] = 'magic',
+    [776] = 'familiar',
+    [1032] = 'healing',
+    [1290] = 'techniques',
+}
+
+local gear = {
+    [27496] = 'Herculean Boots',
+    [27140] = 'Herculean Gloves',
+    [25642] = 'Herculean Helm',
+    [25842] = 'Herculean Trousers',
+    [25718] = 'Herculean Vest',
+    [27497] = 'Merlinic Crackows',
+    [27141] = 'Merlinic Dastanas',
+    [25643] = 'Merlinic Hood',
+    [25719] = 'Merlinic Jubbah',
+    [25843] = 'Merlinic Shalwar',
+    [25716] = 'Odyssean Chestplate',
+    [25840] = 'Odyssean Cuisses',
+    [27138] = 'Odyssean Gauntlets',
+    [27494] = 'Odyssean Greaves',
+    [25640] = 'Odyssean Helm',
+    [25841] = 'Valorous Hose',
+    [27495] = 'Valorous Greaves',
+    [25717] = 'Valorous Mail',
+    [25641] = 'Valorous Mask',
+    [27139] = 'Valorous Mitts',
+    [25720] = 'Chironic Doublet',
+    [27142] = 'Chironic Gloves',
+    [25644] = 'Chironic Hat',
+    [25844] = 'Chironic Hose',
+    [27498] = 'Chironic Slippers',
+    [21754] = 'Aganoshe',
+    [20677] = 'Colada',
+    [20505] = 'Condemners',
+    [21746] = 'Digirbalag',
+    [21072] = 'Gada',
+    [22054] = 'Grioavolr',
+    [22134] = 'Holliday',
+    [21904] = 'Kanaria',
+    [21804] = 'Obschine',
+    [21854] = 'Reienkyo',
+    [20579] = 'Skinflayer',
+    [22113] = 'Teller',
+    [21021] = 'Umaru',
+    [21686] = 'Zulfiqar',
+}
+
+function make_timestamp()
+    return os.date('%Y:%m:%d %H:%M:%S')
+end
+
+local function save_augments()
+    local fh
+    local path = 'data/%s_augments.csv':format(windower.ffxi.get_player().name)
+    if not files.exists(path) then
+        files.create_path('data')
+        fh = io.open(windower.addon_path .. path, 'w')
+        local header = "timestamp,name,stone,path,augment 1,augment 2,augment 3,augment 4,augment 5\r"
+        fh:write(header)
+        fh:close()
+    end
+    fh = io.open(windower.addon_path .. path, 'a')
+    local output = "%s,%s,%s,%s,":format(make_timestamp(),item.name, item.stone, item.path)
+    for i = 1, 4 do
+        output = output.."%s,":format(item.augments.augments[i])
+    end
+    output = output.."%s\r":format(item.augments.augments[5])
+    local fh = io.open(windower.addon_path .. path, 'a')
+    fh:write(output)
+    fh:close()
+end
+
+local function watch_trade(id, org)
+    if id == 0x36 then
+        local p = packets.parse('outgoing', org)
+        local name = (windower.ffxi.get_mob_by_id(p['Target']) or {}).name
+        if (name == 'Oseem') and (p['Number of Items'] == 1) then
+            item = windower.ffxi.get_items(0, p['Item Index 1'])
+        end
+    end
+end
+
+local function broseemify(id, org)
+    if (id == 0x5B) and item.id then
+        local p = packets.parse('outgoing', org)
+        local name = (windower.ffxi.get_mob_by_id(p['Target']) or {}).name
+        if name == 'Oseem' and paths[p['Option Index']] then
+            item.stone = stones[p['_unknown1']]
```

Comment:
You seem to have figured out a previously unknown value for a packet :) If you feel like it you can update our fields file accordingly, to add that info to our library. Unrelated to this PR, just a suggestion.